app/views/admin/admin_users/bulk_search.html.erb

```diff
@@ -0,0 +1,45 @@
+<div class="admin">
+  <!--Descriptive page name, messages and instructions-->
+  <h2 class="heading"><%= ts("Bulk Email Search") %></h2>
+  <!--/descriptions-->
+
+  <!--main content-->
+  <div class="notice">
```

Comment:
We usually put the notice class directly on the paragraph, but other than that, the front end stuff looks good to me!